ansible/deploy-to-node.yaml

```diff
@@ -177,7 +177,7 @@
             "{{ lookup('ansible.builtin.env', 'WEB_DISCORD_CLIENT_ID') }}"
           DISCORD_CLIENT_SECRET:
             "{{ lookup('ansible.builtin.env', 'WEB_DISCORD_CLIENT_SECRET') }}"
-          EMAIL_FROM: open-assistent@laion.ai
```

Comment:
Sorry, we cannot easily change this.

Reply:
@andreaskoepf could you provide some insight on how this should be fixed? cause I was thinking of fixing it the same way @swaptr did

Reply:
@ArgiesDario The problem was not the source change, but creation of the account/alias on the admin side.